src/commands.rs

```diff
@@ -0,0 +1,100 @@
+use clap::Subcommand;
+use strum::{Display as StrumDisplay, EnumString};
+
+/// An enum of commands that need to be passed to pnpm
+#[derive(Debug, PartialEq, EnumString, StrumDisplay, Subcommand)]
+#[strum(ascii_case_insensitive)]
+#[strum(serialize_all = "kebab-case")]
```

Comment:
1. You have to add `#[strum(use_phf)]` to actually use phf.
2. Commands in pnpm is case sensitive. Please remove `ascii_case_insensitive`.
3. I prefer all attributes in a single line.

```suggestion
#[strum(use_phf, serialize_all = "lowercase")]
```

Reply:
shouldnt serialize all be `kebab-case`? otherwise strum would convert SetScript to `setscript` wouldnt it?

Reply:
> shouldnt serialize all be `kebab-case`?

Made a mistake. I copied it from a codebase of my own project. Anyway, I will edit it.